designer/server/src/main/scala/pl/touk/nussknacker/ui/migrations/MigrationService.scala

```diff
@@ -38,76 +47,162 @@ class MigrationService(
     processAuthorizer: AuthorizeProcess,
     processChangeListener: ProcessChangeListener,
     scenarioParametersService: ProcessingTypeDataProvider[_, ScenarioParametersService],
-    useLegacyCreateScenarioApi: Boolean
+    useLegacyCreateScenarioApi: Boolean,
+    migrationApiAdapterService: MigrationApiAdapterService
 )(implicit val ec: ExecutionContext) {
 
   private val passUsernameInMigration = true
 
   def migrate(
-      migrateScenarioRequest: MigrateScenarioRequest
+      migrateScenarioData: MigrateScenarioData
+  )(implicit loggedUser: LoggedUser): EitherT[Future, NuDesignerError, Unit] = {
```

Comment:
IMO we should not use here `NuDesignerError` but the specific for the MigrationService errors hierarchy

Reply:
Ok, done